plone/app/dexterity/tests/editing.rst

```diff
@@ -129,6 +129,7 @@ and saved::
   ...     browser.getControl('Field type').getControl(
   ...         value=factory.title).selected = True
   ...     browser.getControl('Add').click()
+  ...     schema = plonista.lookupSchema()
```

Comment:
Looks good to me, probably `>>> schema = plonista.lookupSchema()` in line 119 can be removed:
https://github.com/plone/plone.app.dexterity/pull/314/files#diff-479ccfb98b9b64526753674cb4ee2672R119